src/plugins/ffmemless/plugin.c

```diff
@@ -585,7 +687,9 @@ static int ffm_sink_prepare(NSinkInterface *iface, NRequest *request)
 		 * keep repeating them until timer runs out and effect ends.
 		 */
 		copy->repeat = INT32_MAX; /* repeat to "infinity" */
-		copy->playback_time = playback_time;
+		if (!copy->playback_time && playback_time) {
```

Comment:
this was resetting playback time to zero on CONSTANT effects, which was not nice.